### DISCUSSION TITLE
Welcome to leetcode-cli Discussions!

### DISCUSSION THREAD

**Discussion by clearloop** [clearloop/leetcode-cli#120] [Announcements]
## 👋 Welcome!
  We’re trying Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

---

Sorry friends, it has been a while I haven't joined the development of this project, really appreciate the contributions you have made in the past years!!!

For me I'm just back to the open source, and getting the project updated is my first work!

Feel free to join the discussions in telegram https://t.me/+U_5si6PhWykxZTI1

**Comment by lucaorofina** [comment]
Hi,

I am having trouble using the commands (leetcode data, leetcode list, anything), I have everything installed on MacOS but there is a problem connecting to leetcode. It says there is a 'site' missing field. Whenever i type one of the commands it gives me:

Config(Error { inner: Error { inner: TomlError { message: "missing field `site`", raw: Some("[code]\neditor = 'emacs'\n# Optional parameter\neditor_args = ['-nw']\n# Optional environment variables (ex. [ \"XDG_DATA_HOME=...\", \"XDG_CONFIG_HOME=...\", \"XDG_STATE_HOME=...\" ])\neditor_envs = []\nlang = 'rust'\nedit_code_marker = false\nstart_marker = \"\"\nend_marker = \"\"\n# if include problem description\ncomment_problem_desc = false\n# comment syntax\ncomment_leading = \"\"\ntest = true\n\n[cookies]\ncsrf = ${my ncrf} '\nsession = '${my sessionID} ' \n\n[storage]\ncache = 'Problems'\ncode = 'code'\nroot = '~/.leetcode'\nscripts = 'scripts'\n"), keys: ["cookies"], span: Some(378..1357) } } })

I have my correct Leetcode cookie data (session and ncsrf) I have been following this tutorial and your READ.me: https://medium.com/steven-gong/how-to-code-on-leetcode-from-the-terminal-b9e1f3b11bba

Thank you
